content/en/blog/2020/istiod/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Introducing istiod: simplifying the control plane"
+subtitle: Consolidating Istio components into ‘istiod’ simplifies mesh operability, while retaining Istio’s powerful functionality
+description: Istiod consolidates the Istio control plane components into a single binary.
+publishdate: 2020-03-19
+attribution: "Craig Box (Google)"
+keywords: [istiod,control plane,operator]
+---
+
+Microservices are a great pattern when they map services to disparate teams that deliver them, or when the value of independent rollout and the value of independent scale are greater than the cost of orchestration. We regularly talk to customers and teams running Istio in the real world, and they told us that none of these were the case for the Istio control plane. So, in Istio 1.5, we've changed how Istio is packaged, consolidating the control plane functionality into a single binary called **`istiod`**.
```

Comment:
I'm not crazy about the backtick version (seems to override the bold, in this case too).

How about this: forget about the backticks and instead use bold lowercase everywhere (**istiod**). We can revisit other pages and do the same there in a followup PR.

You'll also need to add lowercase istiod to .spelling to make the linter happy.
WDYT?